config/flipt.schema.cue

```diff
@@ -142,9 +142,12 @@ import "strings"
  		read_only?: bool | *false
 		local?: path: string | *"."
 		git?: {
-			repository:      string
-			ref?:            string | *"main"
-			poll_interval?:  =~#duration | *"30s"
+			repository:         string
+			ref?:               string | *"main"
+			poll_interval?:     =~#duration | *"30s"
+			ca_cert_path:       string
+			ca_cert_bytes:      string
+			insecure_skip_tls?: bool | false
```

Comment:
If you add a `*` here then CUE will default to `false` (when not supplied).  Otherwise, this is just a type assertion and the `| false` bit is redundent. 
```suggestion
			insecure_skip_tls?: bool | *false
```